src/core_atmosphere/dynamics/mpas_atm_time_integration.F

```diff
@@ -7262,11 +7263,19 @@ subroutine summarize_timestep(domain)
        real (kind=RKIND) :: lonMax, lonMax_global
        real (kind=RKIND), dimension(5) :: localVals, globalVals
 
-       real (kind=RKIND) :: spd
+       real (kind=RKIND), dimension(:,:), allocatable :: spd
+       !$acc declare create(spd)
        real (kind=RKIND), dimension(:,:), pointer :: w
        real (kind=RKIND), dimension(:,:), pointer :: u, v, uReconstructZonal, uReconstructMeridional, uReconstructX, uReconstructY, uReconstructZ
        real (kind=RKIND), dimension(:,:,:), pointer :: scalars, scalars_1, scalars_2
 
+       integer :: lin_indexMax ! Position of min/max in some array as a single integer.
+                               ! If multiple locations match the extrema, always pick the lowest lin_indexMax.
+                               ! array(k,idx) -> lin_indexMax = k+(idx-1)*size(array,1)
+                               ! array(5,4) with size(array,1) = 10 -> lin_indexMax = 35
+
+       logical found_NaN
```

Comment:
I think we need a `::` between `logical` and `found_NaN` for consistency of style.